rules-tests/TypeDeclaration/Rector/Property/PropertyTypeDeclarationRector/Fixture/skip_typed_inherit_doc.php.inc

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Property\PropertyTypeDeclarationRector\Fixture;
+
+class A {
+    /** @var string */
+    public $property;
+}
+
+final class B extends A {
+    /** @inheritDoc */
+    public $property = 'test';
+}
+?>
```

Comment:
Btw, the tests classes names should be unique and descriprive. The fixture from one directory are loaded at once. That might lead to silent autoload collision of using first class name properties.

E.g. here:

```diff
-class A
+class SkipTypedInheritDoc
 {
```